tpchgen/src/generators.rs

```diff
@@ -1229,9 +1284,9 @@ pub struct Order<'a> {
     /// Order date
     pub o_orderdate: TPCHDate,
     /// Order priority
-    pub o_orderpriority: String,
-    /// Clerk who processed the order
-    pub o_clerk: String,
+    pub o_orderpriority: &'a str,
```

Comment:
I suspect these two changes are responsible for most of the performance improvement